test/verifiedTests/testSBML/testReadSBML.m

```diff
@@ -52,7 +32,7 @@
 
 
 % load the models
-modelArr = {'Abiotrophia_defectiva_ATCC_49176.xml', 'STM_v1.0.xml', 'iIT341.xml', 'Ec_iAF1260_flux1.xml'};
+modelArr = { 'STM_v1.0.xml', 'iIT341.xml'};
```

Comment:
I would suggest however to leave `Abiotrophia_defectiva_ATCC_49176.xml'` as a test model.